helm-chart/renku/templates/notebooks/test.yaml

```diff
@@ -113,8 +113,8 @@ spec:
         - name: NB_SESSIONS__SSH__HOST_KEY_SECRET
           value: {{ $.Values.notebooks.ssh.hostKeySecret | quote }}
         {{- end }}
-        - name: NB_CRC_URL
-          value: {{ printf "http://%s/api/data" $.Values.global.crc.serviceName }}
+        - name: NB_DATA_SERVICE_URL
+          value: {{ printf "http://%s/api/data" $.Values.global.dataService.serviceName }}
```

Comment:
You fully removed `.globa.dataService.serviceName` from the values. So better to use the same logic of putting the name together as in notebooks. Or even better make a template for it in `_helpers.tpl` and use the same value everywhere.